sklearn/preprocessing/_polynomial.py

```diff
@@ -642,6 +643,14 @@ class SplineTransformer(TransformerMixin, BaseEstimator):
         Order of output array in the dense case. `'F'` order is faster to compute, but
         may slow down subsequent estimators.
 
+    handle_missing : {'error', 'zeros'}, default='error'
+        Specifies the way missing values are handled.
+
+        - 'error' : Raise an error if np.nan values are present during :meth:`fit`
+        - 'zeros' : Encode missing values as splines with value `0`.
```

Comment:
In terms of API, I think it'll be a lot more consistent to call this `"ignore"` instead of `"zeros"`.

Reply:
We should discuss, if this test should hold or not:

```
    # assert_allclose(
    #    X_nan_fit_transformed.sum(axis=1), np.ones(X_nan.shape[0]) * X_nan.shape[1]
    # )
```

By outputting 0-splines for nans, we change the the output.

Reply:
I thought what we agreed upon, was to return 0's all around, as opposed to return something as if `nan` was a `0`, which I cannot find the purpose of.

Reply:
So in effect we're simply ignoring `nan`s, and hence this parameter fits to be `ignore` instead of `zeros`

Reply:
No, we are not ignoring `nan`s, we invent spline values for them (array of 0s) when we actually don't know what those `nan` values represent.
I would understand ignoring as returning a spline array of `nan`s for every `nan` input. The PR is not adding a functionality to ignore, but to interprete the nans.